waiter/src/waiter/state/maintainer.clj

```diff
@@ -711,7 +725,9 @@
       {:go-chan go-chan
        :notify-instance-killed-fn (fn notify-router-state-maintainer-of-instance-killed [instance]
                                     (log/info "received notification of killed instance" (:id instance))
-                                    (async/go (async/>! kill-notification-chan {:instance instance})))
+                                    (scheduler/log-service-instance instance :kill)
```

Comment:
The event is `kill-notification`. The `kill` has been triggered elsewhere, it should be added in the scheduler implementation.

Reply:
Done